packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/gateway/L1CustomGateway.sol

```diff
@@ -57,15 +57,16 @@ contract L1CustomGateway is L1ArbitrumExtendedGateway, ICustomGateway {
 
     // end of inline reentrancy guard
 
-    function outboundTransfer(
+    function outboundTransferCustomRefund(
```

Comment:
we want to add this in, but also keep the `function outboundTransfer(` override here to add in the `nonReentrant` modifier

Reply:
let's consolidate the discussion in [the other thread](https://github.com/OffchainLabs/arbitrum/pull/2107/files#r850202260)